packages/typescript-estree/tests/lib/parse.test.ts

```diff
@@ -479,6 +494,48 @@ describe('parseAndGenerateServices', () => {
           `);
         });
       });
+
+      describe('"parserOptions.extraFileExtensions" is non-empty and EXPERIMENTAL_useProjectService is true', () => {
```

Comment:
We _just_ applied this rename recently ✨ 

```suggestion
      describe('"parserOptions.extraFileExtensions" is non-empty and projectService is true', () => {
```